src/player/controller.rs

```diff
@@ -88,8 +84,8 @@ pub fn update_player_controllers(world: &mut World) {
                 let input = collect_local_input(*input_scheme);
                 controller.apply_input(input);
             }
-            PlayerControllerKind::Network(_account_id) => {
-                // TODO: Network input
+            PlayerControllerKind::Network(_player_id) => {
+                // not implemented
```

Comment:
How about using [` unimplemented!();`](https://doc.rust-lang.org/std/macro.unimplemented.html)? 🐻

Reply:
nah, I am currently working on that part, I just had to remove some code so I wouldn't have to branch to add the has...
It won't be triggered any ways, so no need for a panic there